ipatests: add test_trust suite to nightly runs

wladich opened the pull request:
The test suite test_trust was missing in nightly definitions
because PR-CI was not able to provision multi-AD topology.
Now that PR-CI is updated, we can start executing this test suite.
It is not reasonable to add it to gating as this suite is
time consuming like other tests requiring provisioning of AD instances.

abbra commented:
LGTM. Please remove the temp commit.

flo-renaud commented on ipatests/prci_definitions/nightly_rawhide.yaml:
```diff
@@ -1570,3 +1574,15 @@ jobs:
+    job:
+      class: RunADTests
+      args:
+        build_url: '{fedora-rawhide/build_url}'
```
`update_packages: True` is also needed for rawhide tests.

wladich commented:
Added "update_packages: True" for rawhide definition
Also added same line  in nightly_latest_testing
Also added same line for test_winsyncmigrate in both files.

abbra commented:
@wladich looks like you need to rebase this PR now that I pushed bunch of others. It was unclear to me why backports were created while this one wasn't pushed. I pushed backports and other PRs first and now there is a conflict:
```
Applying to master: ipatests: add test_trust suite to nightly runs
git am
Applying: ipatests: add test_trust suite to nightly runs
Patch failed at 0001 ipatests: add test_trust suite to nightly runs
When you have resolved this problem, run "git am --continue".
If you prefer to skip this patch, run "git am --skip" instead.
To restore the original branch and stop patching, run "git am --abort".
error: patch failed: ipatests/prci_definitions/nightly_latest_testing.yaml:1551
error: ipatests/prci_definitions/nightly_latest_testing.yaml: patch does not apply
error: patch failed: ipatests/prci_definitions/nightly_rawhide.yaml:1566
error: ipatests/prci_definitions/nightly_rawhide.yaml: patch does not apply
hint: Use 'git am --show-current-patch' to see the failed patch
→ 128
Command failed

```

wladich commented:
Rebased without conflicts.

wladich commented:
master:

* 001de6eed70d96a3e1e12554e8320d8f18ebf5a6 ipatests: add test_trust suite to nightly runs
